haskell.html.markdown

```diff
@@ -198,11 +198,11 @@ foo 5 -- 15
 -- multiplies the result of that by 4, and then returns the final value.
 foo = (4*) . (10+)
 
--- 4*(10 + 5) = 60
+-- 4*(10+ 5) = 60
```

Comment:
Sure - either is fine, but my intention here was to help the reader match the sections exactly (to stress that "4*" and "10+" can be seen as unary functions).